src/components/Widget.tsx

```diff
@@ -143,7 +143,7 @@ export function TestableWidget(props: PropsWithChildren<TestableWidgetProps>) {
     }
     return props.defaultChainId
   }, [props.defaultChainId])
-  const jsonRpcUrlMap: string | JsonRpcProvider | { [chainId: number]: string[] } = useMemo(() => {
+  const jsonRpcUrlMap: { [chainId: number]: string[] } = useMemo(() => {
```

Comment:
would prob define this type for jsonRpcUrlMap above somewhere

Reply:
I'm going to omit this type because it's actually enforced when passed to `ActiveWeb3Provider`.